internal/aws/containerinsight/const.go

```diff
@@ -26,6 +26,7 @@ const (
 	MetricType              = "Type"
 	SourcesKey              = "Sources"
 	Timestamp               = "Timestamp"
+	OSType                  = "os"
```

Comment:
should this be more explicit? OperatingSystem?

Reply:
Addressed